conf/application.conf

```
@@ -87,6 +87,8 @@ features {
   exportTiffMaxVolumeMVx = 1024
   exportTiffMaxEdgeLengthVx = 8192
   defaultToLegacyBindings = false
+  # The Only valid item value is currently "ConnectomeView":
+  optInTabs = ["ConnectomeView"]
```

remove before merging
```suggestion
  optInTabs = []
```